File: app/src/main/java/Model/UserModel.java
package Model;

import com.google.firebase.database.DatabaseReference;
import com.google.firebase.database.FirebaseDatabase;

public class UserModel {
    private String id;
    private String nome;
    private String email;
    private String senha;

    public UserModel() {}

    public UserModel(String id, String nome, String email, String senha) {
        this.id = id;
        this.nome = nome;
        this.email = email;
        this.senha = senha;
    }

    public String getId() { return id; }
    public void setId(String id) { this.id = id; }

    public String getNome() { return nome; }
    public void setNome(String nome) { this.nome = nome; }

    public String getEmail() { return email; }
    public void setEmail(String email) { this.email = email; }

    public String getSenha() { return senha; }
    public void setSenha(String senha) { this.senha = senha; }

    public void salvar() {
        DatabaseReference reference = FirebaseDatabase.getInstance().getReference();
        reference.child("usuarios").child(id).setValue(this);
    }
}

File: app/src/main/java/com/univap/lume/model/UserModel.kt
package com.univap.lume.model
import com.google.firebase.database.FirebaseDatabase

data class UserModel(
    var id: String = "",
    var nome: String = "",
    var email: String = ""
) {
    fun salvar() {
        val ref = FirebaseDatabase.getInstance().reference
        ref.child("usuarios")
            .child(id)
            .setValue(this)
    }
}